samples/QuickstartWeatherServer/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol;
+
+var builder = Host.CreateEmptyApplicationBuilder(settings: null);
+
+//builder.Logging.ClearProviders();
+//builder.Logging.AddFilter("Microsoft", LogLevel.Warning); // Adjust the log level as needed
+
+builder.Services.AddMcpServer()
+    .WithStdioServerTransport()
+    .WithToolsFromAssembly();

```

Comment:
WithToolsFromAssembly isn't native aot friendly. That's ok, but I wonder if our getting started samples should err on the side of using native aot friendly constructs? e.g. changing this to be `.WithTools<WeatherTools>()`

Reply:
We would then have to make `WeatherTools` non-static, so it's a question on what we see as the more common/desired pattern.

Reply:
I think making it non-static is better.

Reply:
Is that going to be the pattern that we most commonly document elsewhere? The quick start should docus on the things that you are most likely to do and the pattern you most likely going to tackle, which is why I went with the non native aot approach.

Reply:
For the simple and common case where you just have a single type, maybe even in your single Program.cs, naming it as part of a `WithTools<T>` call seems like a good answer: it's unambiguous, it's declarative, it's efficient, and it works everywhere.  But in the name of forward progress, we can keep it as you have it for now. Thanks.